.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Build and Deploy b-l475e-iot01a",
+            "type": "shell",
+            "command": "openocd -f board/stm32l4discovery.cfg -c \"program b-l475e-iot01a/demos/projects/ST/b-l475e-iot01a/iot-middleware-sample.elf verify\" -c \"reset halt\" -c shutdown",
+            "windows": {
+                "options": {
+                  "shell": {
+                      "executable": "cmd.exe",
+                      "args": ["/c"]
+                  }
+                }
+              },
+            "problemMatcher": [],
+            "dependsOn": [ "Build b-l475e-iot01a" ]
+        },
+        {
+            "label": "Download FreeRTOS",
+            "type": "shell",
+            "command": "[ -d libs/FreeRTOS ] || ( git clone https://github.com/FreeRTOS/FreeRTOS.git libs/FreeRTOS && cd libs/FreeRTOS && git checkout -b c8fa483b68c6c1149c2a7a8bc1e901b38860ec9b && git submodule update --init --recursive --depth=1 )"
```

Comment:
nit: should we have some kind of script that does this? the step in cmake I think could also just execute the script instead of a built up command? just to reduce duplication and git tags in multiple places etc

Reply:
added